lib/unleash/repo.ex

```diff
@@ -1,124 +1,128 @@
 defmodule Unleash.Repo do
-  @moduledoc false
+  @moduledoc """
+  This genserver polls the unleash service each time the given interval has
+  elapsed, refreshing both our local ETS cache and the backup state file if the
+  flag state has diverged.
+
+  The following configuration values are used:
+
+  Config.features_period(): polling interval - default 15 seconds
+  Config.retries(): number of time the call to refresh values is allowed to retry - default -1 (0)
+  """
+
   use GenServer
   require Logger
 
   alias Unleash.Config
   alias Unleash.Features
+  alias Unleash.Cache
 
   def init(%Features{} = features) do
```

Comment:
in what scenario is `%Features{}` the argument for init?

Reply:
It's used to pre-populate the cache in tests (e.g [here](https://github.com/duffelhq/unleashed_ex/blob/main/test/unleash_test.exs#L144) or [here](https://github.com/duffelhq/unleashed_ex/blob/main/test/unleash/variant_test.exs#L17) or [here](https://github.com/duffelhq/unleashed_ex/blob/main/test/unleash/client_specification_test.exs#L29)